sdk/network/Azure.ResourceManager.Network/tests/Tests/ApplicationGatewayTests.cs

```diff
@@ -517,7 +517,7 @@ public async Task ApplicationGatewayApiTest()
             string resourceGroupName = Recording.GenerateAssetName("csmrg");
 
             string location = "West US";
```

Comment:
Is `West US` intentional? If not, I suggest to use `CreateResourceGroup(resourceGroupName)` which will use the `location` property in the `TestEnviornment`.